tools/project_template/ModuleTemplate/trace.cpp

```diff
@@ -2,25 +2,28 @@
 #include "trace.h"
 
 TRACELOGGING_DEFINE_PROVIDER(
-  g_hProvider,
-  "Microsoft.PowerToys",
-  // {38e8889b-9731-53f5-e901-e8a7c1753074}
-  (0x38e8889b, 0x9731, 0x53f5, 0xe9, 0x01, 0xe8, 0xa7, 0xc1, 0x75, 0x30, 0x74),
-  TraceLoggingOptionProjectTelemetry());
+    g_hProvider,
+    "Microsoft.PowerToys",
+    // {38e8889b-9731-53f5-e901-e8a7c1753074}
+    (0x38e8889b, 0x9731, 0x53f5, 0xe9, 0x01, 0xe8, 0xa7, 0xc1, 0x75, 0x30, 0x74),
+    TraceLoggingOptionProjectTelemetry());
 
-void Trace::RegisterProvider() {
-  TraceLoggingRegister(g_hProvider);
+void Trace::RegisterProvider()
+{
+    TraceLoggingRegister(g_hProvider);
 }
 
-void Trace::UnregisterProvider() {
-  TraceLoggingUnregister(g_hProvider);
+void Trace::UnregisterProvider()
+{
+    TraceLoggingUnregister(g_hProvider);
 }
 
-void Trace::MyEvent() {
-  TraceLoggingWrite(
-    g_hProvider,
-    "PowerToyName::Event::MyEvent",
-    ProjectTelemetryPrivacyDataTag(ProjectTelemetryTag_ProductAndServicePerformance),
-    TraceLoggingBoolean(TRUE, "UTCReplace_AppSessionGuid"),
-    TraceLoggingKeyword(PROJECT_KEYWORD_MEASURE));
+void Trace::MyEvent()
+{
+    TraceLoggingWrite(
+        g_hProvider,
+        "PowerToyName::Event::MyEvent",
```

Comment:
We need to replace the `::` with underscore, I should have mentioned it in the issue.